tests/model_serving/model_server/authentication/test_kserve_token_authentication_raw.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from tests.model_serving.model_server.utils import verify_inference_response
+from utilities.constants import ModelFormat, ModelStoragePath, Protocols, ModelInferenceRuntime
+from utilities.inference_utils import Inference
+
+pytestmark = pytest.mark.usefixtures("valid_aws_config")
+
+
+@pytest.mark.serverless
+@pytest.mark.parametrize(
+    "model_namespace, s3_models_storage_uri",
+    [
+        pytest.param(
+            {"name": "kserve-token-authentication"},
+            {"model-dir": ModelStoragePath.FLAN_T5_SMALL},
+        )
+    ],
+    indirect=True,
+)
+class TestKserveTokenAuthenticationRaw:
```

Comment:
let's split to rest and grpc; then test cross-domain with one protocol. this way we can re-use existing fixtures and not add new ones.